data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/GetEventTypeExpressionFunctionTest.java

```diff
@@ -0,0 +1,55 @@
+package org.opensearch.dataprepper.expression;
+
+import org.opensearch.dataprepper.model.event.Event;
+import org.opensearch.dataprepper.model.event.JacksonEvent;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+class GetEventTypeExpressionFunctionTest {
+
+    private GetEventTypeExpressionFunction createObjectUnderTest() {
+        return new GetEventTypeExpressionFunction();
+    }
+
+    private Event createTestEvent(final String eventType) {
+        return JacksonEvent.builder()
+                .withEventType(eventType)
+                .withData(Map.of())
+                .build();
+    }
+
+    @Test
+    void testGetEventTypeReturnsCorrectType() {
+        GetEventTypeExpressionFunction function = createObjectUnderTest();
+        Event testEvent = createTestEvent("LOG");
```

Comment:
Make this an `@ParameterizedTest` with multiple event types. e.g. `LOG`, `METRIC`, `TRACE`.

Reply:
updated: [01c342c](https://github.com/opensearch-project/data-prepper/pull/5686/commits/01c342c3e851991611da85ea2494850d69579bfd)